src/lib/rule2B.ts

```diff
@@ -0,0 +1,58 @@
+import {computeTextAlternative} from './compute_text_alternative';
+import {Context} from './context';
+
+/**
+ * Get any HTMLElement referenced in the aria-labelledby attribute
+ * of 'elem' that exist in the document (i.e is 'valid')
+ * @param elem - element whose aria-labelledby attribute is considered
+ * @return - An array of any HTMLElement in the document that is referenced
+ * by elem's aria-labelledby
+ */
+function resolveValidAriaLabelledbyIdrefs(elem: HTMLElement): HTMLElement[] {
+  // Get a list of idref strings
```

Comment:
I don't think those comments add much to the code.

comments can get out of date with the code, so it's better to try to make the code readable directly. Please remove all the comments that can easily be inferred from the code